tests/resource/test_resource.py

```diff
@@ -202,8 +202,8 @@ def resources(self):
         assert CustomOPWithResources(wires=[0, 1])  # shouldn't raise an error
 
 
-def _construct_tape_from_ops(lst_ops):
-    with QuantumTape() as tape:
+def _construct_tape_from_ops(lst_ops, shots=None):
+    with QuantumTape(shots=shots) as tape:
         for op in lst_ops:
             qml.apply(op)
     return tape
```

Comment:
something wrong with just `QuantumScript(lst_ops, shots=shots)`?

Reply:
I'm not sure, this is a fairly new test that still uses QuantumTape, so I didn't think it needs to be changed since the change was probably deliberate. @Jaybsoni can confirm.